source/index.ts

```diff
@@ -360,6 +360,11 @@ is.nonEmptySet = <T = unknown>(value: unknown): value is Set<T> => is.set(value)
 is.emptyMap = (value: unknown): value is Map<never, never> => is.map(value) && value.size === 0;
 is.nonEmptyMap = <Key = unknown, Value = unknown>(value: unknown): value is Map<Key, Value> => is.map(value) && value.size > 0;
 
+/**
+Returns `true` if `value` can be used as an object property key (either a `string`, a `number` or a `symbol`).
```

Comment:
@sindresorhus I may have deleted your comment by mistake 🤭 sorry about that.

@PopGoesTheWza His comment was about the comment being like others.

IMO a more fitting comment would be to describe what a `PropertyKey` is, instead of the whole "returns true if...", that's good for the readme but unnecessary in the code IMO, since the reason for the comment here is that it's not clear what `PropertyKey` is at first glance.

Reply:
Something like this?

> Checks if `value` safe to use as a property key (typically when working with `Object`, `Proxy` and `Reflect` methods)

Reply:
Don't think it's needs to go that far, just to explain what `PropertyKey` is, something like:

> `PropertyKey` is any value that can be used as an object key (string, number, or symbol)